NewTest.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- Here we import the Boostrap CSS stylesheet -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/main2.css">
+    <title>Domenico DC - Responsive grid project</title>
+</head>
+<style>
+body{margin: 30px;}
+   .my-container{
+    border: 3px solid green;
+}
+.my-row{
+    border: 3px solid red;
+    height: 300px;
+}
+.my-col{
+    border: 3px dotted blue;
+}
+</style>
```

Comment:
Don't include CSS in your HTML files, it's best practice to have them in separate files. I realise this might just be you experimenting 😉